Marlin/ultralcd.cpp

```diff
@@ -3185,7 +3185,7 @@ void kill_screen(const char* lcd_msg) {
       currentScreen = menu_edit_callback_ ## _name; \
       callbackFunc = callback; \
     } \
-    void _menu_action_setting_edit_accessor_ ## _name (const char * const pstr, _type (*pget)(), void (*pset)(_type), const _type minValue, const _type maxValue) { \
+    void _menu_action_setting_edit_accessor_ ## _name (const char * const pstr, _type (*pget)(), void (*pset), const _type minValue, const _type maxValue) { \
```

Comment:
OK.  Would you like me to update the PR to remove this change, or just close the whole thing and you can commit the other two changes if you're happy with them?

BTW, since creating this I figured out why I was seeing these 'type conversion' errors that no one else was - my environment was missing the -fpermissive compiler flag.  Since adding that I no longer see the errors, but the invalid type conversions are still there if compiler warnings are enabled.